.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release csvcubed-models
+on:
+  release:
+    types: [published]
+
+jobs:
+  test_in_environments:
+    strategy:
+      matrix:
+        # Test in all supported versions of python in both Windows and Ubuntu.
+        python-version: ['3.9', '3.10', '3.11']
+        os: [ubuntu-latest, windows-latest]
+    uses: ./.github/workflows/reusable-test.yaml
+    with:
+      os: ${{ matrix.os }}
+      python-version: ${{ matrix.python-version }}
+
+  build_package:
+    if: needs.test_in_environments.result == 'success'
```

Comment:
This is going to be run concurrently with `reuseable-test.yaml` since it has its own triggers.